frontend/lib/src/components/elements/CodeBlock/StreamlitSyntaxHighlighter.tsx

```diff
@@ -29,11 +32,13 @@ export interface StreamlitSyntaxHighlighterProps {
   children: string | string[]
   language?: string
   showLineNumbers?: boolean
+  wrapLines?: boolean | undefined
```

Comment:
nit: I think the `undefined` is not needed here since it is also implied by the `?`

Reply:
Removed. I only had that because the rest of the code had it too (although now it's been removed)